### PR TITLE
Fix some default settings

### PR DESCRIPTION
**Pull request by piebrain**
Sets default radio channel to 94.7MHz in fixed tune mode if there is no config file on the device.

@sastel please test on your device without config file.

### CODE REVIEW

**Comment by sastel** [comment]
@piebrain it isn't working. Is there something I need to do besides loading it to the device?
``` 
 236.        PLAY got MODE_BTN_EVT
 237.      PLAYER got MODE_BTN_EVT
 238.      PLAYER going to FM_RADIO  
 239.        PLAY got EXIT_EVT  
 240.      PLAYER got EXIT_EVT  
 241.    FM_RADIO got ENTRY_EVT 
Waited for CTS for 32 ms

Radio volume set to 41
Waited for CTS for 2 ms
Set to channel: 0
 242.    FM_RADIO got START_EVT 
 243.    FM_RADIO going to SLEEP     
 244.       SLEEP got ENTRY_EVT 
 245.       SLEEP got START_EVT 

sleeping
force slp ena
```

**Comment by piebrain** [comment]
@sastel maybe you can test again

**Comment by sastel** [comment]
@piebrain you got it.
I'll review.

File: lib/Compass/Settings.cpp [hunk 1]
```diff
@@ -43,6 +43,11 @@ void SettingsClass::saveTimestamp(long ts) {
     tSD.close();
 }
 
+void SettingsClass::loadDefaults() {
+    fixedFreqs[0] = DEFAULT_FM_CHANNEL;
+    fixedCount = 1;
```
Comment (sastel) [inline lib/Compass/Settings.cpp:48]: Maybe this should be renamed "fixedFreqCount".

**Review by sastel — APPROVED** [review]
Looks good. Ship it! :)

**Pull request merged**